src/standalone.js

```diff
@@ -232,7 +231,13 @@ export default async function(initialOptions) {
   const result = {};
 
   result.glyphsData = await getGlyphsData(filteredFiles, options);
-  result.glyphsData.map(options.beforeFontGenerationGlyphTransformation);
+  result.glyphsData = result.glyphsData.map(glyphData => {
+    if (typeof options.glyphTransformFn === "function") {
+      glyphData.metadata = options.glyphTransformFn(glyphData.metadata);
```

Comment:
Awesome work :+1: Maybe we can implement function as async too (i think some people can store order in other file)?

Reply:
Sure, I've been using a json file with a map that is updated every time new icons are build. But I'll make it async.

Reply:
:+1: Let's do it and make release after merge